pkg/gateway/gateway.go

```diff
@@ -338,6 +341,10 @@ func (g *gatewayType) onStoppedLeading(ctx context.Context) {
 
 	g.cableEngine.Stop()
 
+	if g.SigningRequestor != nil {
+		_ = g.SigningRequestor.Uninstall(ctx)
+	}
```

Comment:
We want to `Uninstall` here as it will remove all CSRs. This really should be done in `uninstall`.